<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Shashwat" ignored="0" name="Test-Online_Enrollment-US" tests="2" failures="1" timestamp="2021-09-28T19:08:44 IST" time="192.873" errors="0">
  <testcase name="createGuestOrder" time="90.831" classname="framework.test.OnlineEnrollment_US_PartyLite">
    <skipped/>
  </testcase> <!-- createGuestOrder -->
  <testcase name="createGuestOrder" time="85.716" classname="framework.test.OnlineEnrollment_US_PartyLite">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
at framework.utils.Reports.reportStep(Reports.java:72)
at framework.utils.Reports.reportStep(Reports.java:82)
at framework.test.OnlineEnrollment_US_PartyLite.createGuestOrder(OnlineEnrollment_US_PartyLite.java:108)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 18 stack frames]]>
    </failure>
  </testcase> <!-- createGuestOrder -->
</testsuite> <!-- Test-Online_Enrollment-US -->
